ApiReview/API.Polly.Core/NoDocs/Polly.Registry/ConfigureBuilderContext.cs

```diff
@@ -0,0 +1,15 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System.Threading;
+
+namespace Polly.Registry;
+
+public class ConfigureBuilderContext<TKey> where TKey : notnull
+{
+    public TKey PipelineKey { get; }
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public void EnableReloads(Func<Func<CancellationToken>> tokenProducerFactory);
```

Comment:
```suggestion
    public void AddReloadToken(CancellationToken cancellationToken);
```
Would simplify the API and allows to add multiple reload sources to the context.